app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -305,8 +305,6 @@ export const ContentListView: React.FC = () => {
                   selected={
                     !!filterAdvanced.startDate ? new Date(filterAdvanced.startDate) : undefined
```

Comment:
Did you test this to determine what time it does set?  I'm wondering if it will ignore content that is either at the beginning or later in the day depending on the time value.

Reply:
Defaults to 12am!

Reply:
To confirm: 

- from time starts at 12 AM (00:00:00) 
- to time now is 11:59 PM (23:59:59)